Makefile

```diff
@@ -71,12 +71,9 @@ all: generate lint manager bin/loki-broker
 OCI_RUNTIME ?= $(shell which podman || which docker)
 
 # Run tests
-ENVTEST_ASSETS_DIR=$(CURDIR)/testbin
 test: generate go-generate lint manifests
 test: $(GO_FILES)
-	@mkdir -p ${ENVTEST_ASSETS_DIR}
-	@test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.7.0/hack/setup-envtest.sh
-	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
+	go test ./... -coverprofile cover.out
```

Comment:
Did you mean to delete these? I am happy to see them gone, but what outcomes should we expect? I havn't investigated the effects of this.